src/Processor/UpdatedOrderProcessor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Processor;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Setono\SyliusOrderEditPlugin\Exception\NewOrderWrongTotalException;
+use Sylius\Component\Core\Inventory\Operator\OrderInventoryOperatorInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\Processor\OrderProcessorInterface;
+
+final class UpdatedOrderProcessor
```

Comment:
Missing interface

Reply:
I was not sure about your approach for interfaces - and probably I spent too much time lastly on the end-applications not on the packages 😅 I will add one 🫡